libs/core/src/graphql/build-fragment-spread.ts

```diff
@@ -0,0 +1,10 @@
+import { FragmentDefinitionNode } from 'graphql'
+
+/**
+ * Builds a string of fragment names that can be interpolated into a GraphQL query.
+ * Each name is separated by a newline character: '\n'.
+ * If an empty array is passed, an empty string is returned.
+ * @param names A list of fragment names.
+ */
+export const daffBuildFragmentSpread = (names: FragmentDefinitionNode['name']['value'][]): string =>
```

Comment:
This arg type should likely be expanded to `FragmentDefinitionNode[]`

Reply:
okay but I think I'll make it a rest param so that single fragments can be passed without having to create an array.

Reply:
changed